CONTRIBUTING.md

```diff
@@ -189,6 +189,44 @@ Take a look at the sections that follow to pick a route (or five) that appeal to
 
 If you ever find yourself at a loss for what to do, or in need of mentorship or advice on how to contribute to Bevy, feel free to ask in [Discord] and one of our more experienced community members will be happy to help.
 
+### Join a working group
+
+Active initiatives in Bevy are organized into temporary working groups: choosing one of those and asking how to help can be a fantastic way to get up to speed and be immediately useful.
+
+Working groups are public, open-membership groups that work together to tackle a broad-but-scoped initiative.
+The work that they do is coordinated in a forum-channel on [Discord](https://discord.gg/bevy), although they also create issues and may use project boards for tangible work that needs to be done.
+
+There are no special requirements to be a member, and no formal membership list.
+Working groups are *spaces*, not clubs.
+
+### Start a working group
+
+When tackling a complex initiative, friends and allies can make things go much more smoothly.
+
+To start a working group:
+
+1. Decide what the working group is going to focus on. This should be tightly focused and achievable!
+2. Gather at least 3 people including yourself who are willing to be in the working group.
+3. Ping the `@Maintainer` role on Discord in #engine-dev announcing your mutual intent.
+
+One of the maintainers will then create a forum channel for you, as a purely administrative task.
```

Comment:
I think a working group as defined here is a "soft endorsement" by the project signaling that we would like to help facilitate collaboration on and direct attention to a specific effort. Adding a new working group adds it to the pile of existing working groups, which increases competition for community attention. Creating a working group for something we know is unmergable (or cannot reasonably be prioritized or adopted by the project at this point in time) would be a waste of everyone's time.

I think there should be a step (4), where maintainers and relevant SMEs discuss whether or not the general "idea" behind the working group is compatible with Bevy and worth us officially directing effort and attention to it. I don't think this needs to be particularly strict from a technical perspective. Just broad strokes answers to the questions (1) is this something Bevy should have and (2) is this the right time to start doing it?

Reply:
To be clear, I don't think a "working group proposal" needs to be anything more than a few sentences specifying the general direction.